src/utility/tests/utility_function_test.cc

```diff
@@ -0,0 +1,47 @@
+#include "../utility_functions.h"
+
+#include <algorithm>
+#include <array>
+#include <iostream>
+#include <numeric>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+#include "../../test_helpers/gmock_wrapper.h"
```

Comment:
The header files developed in BART should always be included in the first block if there's not any technical difficulty doing it.